dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/VersionDAO.java

```diff
@@ -44,4 +45,11 @@ public void delete(T version) {
     public Version<T> findVersionInEntry(Long entryId, Long versionId) {
         return uniqueResult(namedQuery("io.dockstore.webservice.core.Version.findVersionInEntry").setParameter("entryId", entryId).setParameter("versionId", versionId));
     }
+
+    // Currently not used for anything, will be used for paginated versions
+    public Long getVersionsCount(Long entryId) {
```

Comment:
ditto for parameter
for the return type, might want to consider `Optional<Long>` to make the caller handle the case when the result is null (the entry doesn't exist) which is distinct from 0 (entry exists, but legitimately has no versions)